sdk/js/.eslintrc.json

```diff
@@ -9,7 +9,11 @@
                 "@nx/dependency-checks": [
                     "error",
                     {
-                        "ignoredDependencies": []
+                        "ignoredDependencies": [
+                            "fetch-retry",
```

Comment:
you shouldnt have to ignore any of these now I think it should pick up their usage

Reply:
probably right, a lot of this got lost in translation with my 1.4 billion iterations to fix the build. Ill test it out